tw-tkms-starter/src/main/java/com/transferwise/kafka/tkms/TkmsTopicValidator.java

```diff
@@ -92,7 +92,7 @@ public void preValidateAll() {
       var shardProperties = tkmsProperties.getShards().get(shard);
       List<String> shardTopics = shardProperties == null ? null : shardProperties.getTopics();
 
-      if (shardTopics != null && shard == tkmsProperties.getDefaultShard()) {
+      if (shardTopics != null && !shardTopics.isEmpty() && shard == tkmsProperties.getDefaultShard()) {
```

Comment:
Looks fine to me but shouldn't there be an associated test to cover this bug?

Reply:
Yep, test coverage was added by changing the test config.

Reply:
ahh yes, sorry missed that.